Classes/Private/Manager/Database/OCTDBManager.m

```diff
@@ -231,6 +231,29 @@ - (void)removeChatWithAllMessages:(OCTDBChat *)chat
     });
 }
 
+#pragma mark - Calls
+- (RLMResults *)allCalls
+{
+    __block RLMResults *results;
+
+    dispatch_sync(self.queue, ^{
+        results = [OCTDBCall allObjectsInRealm:self.realm];
+    });
+
+    return results;
+}
+
+- (RLMResults *)callsWithChat:(OCTDBChat *)chat
+{
+    __block RLMResults *results;
+
+    dispatch_sync(self.queue, ^{
+        results = [OCTDBCall objectsInRealm:self.realm where:@"uniqueIdentifier == %@", chat.uniqueIdentifier];
```

Comment:
I don't think that uniqueIdentifier should be same as for chat. Also there is no need in uniqueIdentifier at all, you can just use `where:@"chat == %@", chat];`